### DISCUSSION TITLE
Cluster Bar Chart Support

### DISCUSSION THREAD

**Discussion by reyesGeorge** [recharts/recharts#3034] [Ideas]
Are you guys planning on adding support for Cluster Bar Charts? I made my own custom version of some of the charts, however it is hard to add unique customization to each bar when I hover over it.

**Comment by Sholls000** [comment]
Hi George, Could you please share tips on how you made the bars clustered @reyesGeorge ?

**Reply by reyesGeorge** [reply]
Yes, the charts intake a list of objects to display data. I made reusable functions connected to restful endpoints that add to multiple objects making up the categories. For example you have five categories, five data sources, you flatten the data and insert to the final object. You can dynamically change the label of the bars, so I used that plus some joins in the background to filter my data by date and display cluster 'x' by the data label, displaying the 5 categories under 'x'. Essentially nest all of your categories/objects under one object. There is a limitation here, where the tooltip is not as informational if you use it.

**Reply by Sholls000** [reply]
Could you please create a working sample with fiddle, so, I could follow your implementation?
